<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Simulation</class>
 <widget class="QWidget" name="Simulation">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1381</width>
    <height>805</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel{
	border-style:solid;
	border-color:red;	
	border-radius:2px;
	border-width:1px;
	background-color:rgb(255, 219, 12);
	
}
	
</string>
  </property>
  <widget class="QLabel" name="lblProc5">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>210</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Proc5</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblProc">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>120</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>proc</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblProc4">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>210</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>proc4</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblProc3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>160</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>proc3</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>160</y>
     <width>371</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color:red;
border-width:2px 0px 2px 0px;
border-style:solid;
border-radius:10px;

</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_9" native="true">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>440</y>
     <width>591</width>
     <height>301</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color:red;
border-width:2px ;
border-style:solid;
border-radius:10px;

</string>
   </property>
   <widget class="QWidget" name="widget_8" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>251</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/icon/cpu.png);
background-color: rgb(32, 237, 255);</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_5" native="true">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>251</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/icon/cpu.png);
background-color: rgb(27, 207, 171)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>260</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>User Mode</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>260</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Kernel Mode</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="prcIcon" native="true">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>140</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/icon/proc4.png);
border-width:0px;</string>
   </property>
  </widget>
  <widget class="QWidget" name="prc3Icon" native="true">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>180</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/icon/proc5.png);
border-width:0px;</string>
   </property>
  </widget>
  <widget class="QWidget" name="prc5Icon" native="true">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>230</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/icon/proc3.png);
border-width:0px;</string>
   </property>
  </widget>
  <widget class="QWidget" name="prc4Icon" native="true">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>230</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/icon/asd.png);
border-width:0px;</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_10" native="true">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>40</y>
     <width>121</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/icon/ram.png);</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_11" native="true">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>650</y>
     <width>311</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color:red;
border-width:2px 0px 2px 2px;
border-style:solid;
border-radius:10px;

</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>260</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color:red;
border-width:2px ;
border-style:solid;
border-radius:10px;
</string>
   </property>
   <property name="text">
    <string>Ready queue</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>750</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color:red;
border-width:2px ;
border-style:solid;
border-radius:10px;
</string>
   </property>
   <property name="text">
    <string>Block queue</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_13" native="true">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>500</y>
     <width>311</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color:red;
border-width:2px 0px 2px 2px;
border-style:solid;
border-radius:10px;

</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>600</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color:red;
border-width:2px ;
border-style:solid;
border-radius:10px;
</string>
   </property>
   <property name="text">
    <string>Suspend</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_14" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>630</y>
     <width>121</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/icon/usb-hd-driver-icon-2.png);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>750</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color:red;
border-width:2px ;
border-style:solid;
border-radius:10px;

</string>
   </property>
   <property name="text">
    <string>Running</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_12" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>490</y>
     <width>121</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/icon/io.png);</string>
   </property>
  </widget>
  <widget class="QWidget" name="prc2Icon" native="true">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>140</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image:url(:/icon/proc1.png);
border-width:0px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblProc2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>120</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Proc2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>760</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>760</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QFrame" name="dispach">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>60</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/time/dispach.png);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>760</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>310</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color:red;
border-width:2px ;
border-style:solid;
border-radius:10px;
</string>
   </property>
   <property name="text">
    <string>Creation</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>760</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Del</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="TimeTbl">
   <property name="geometry">
    <rect>
     <x>1130</x>
     <y>10</y>
     <width>221</width>
     <height>141</height>
    </rect>
   </property>
   <property name="rowCount">
    <number>0</number>
   </property>
   <column>
    <property name="text">
     <string>ProcessId</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Time Slice</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="tblInfo">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>301</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>ProcessId</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Ram</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Mode</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Score</string>
    </property>
   </column>
  </widget>
  <widget class="QWidget" name="Terminate" native="true">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>249</y>
     <width>131</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/Delete/Terminate.png);</string>
   </property>
   <zorder>prc2Icon</zorder>
  </widget>
  <zorder>lblProc5</zorder>
  <zorder>lblProc</zorder>
  <zorder>lblProc4</zorder>
  <zorder>lblProc3</zorder>
  <zorder>widget_2</zorder>
  <zorder>widget_9</zorder>
  <zorder>prcIcon</zorder>
  <zorder>prc3Icon</zorder>
  <zorder>prc5Icon</zorder>
  <zorder>prc4Icon</zorder>
  <zorder>widget_10</zorder>
  <zorder>widget_11</zorder>
  <zorder>label_8</zorder>
  <zorder>label_9</zorder>
  <zorder>widget_13</zorder>
  <zorder>label_10</zorder>
  <zorder>widget_14</zorder>
  <zorder>label_11</zorder>
  <zorder>widget_12</zorder>
  <zorder>lblProc2</zorder>
  <zorder>pushButton</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>dispach</zorder>
  <zorder>pushButton_3</zorder>
  <zorder>label_12</zorder>
  <zorder>pushButton_4</zorder>
  <zorder>TimeTbl</zorder>
  <zorder>Terminate</zorder>
  <zorder>prc2Icon</zorder>
  <zorder>tblInfo</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
